cpp/include/cudf/lists/contains.hpp

```diff
@@ -129,6 +129,7 @@ enum class duplicate_find_option : int32_t {
  * @throw cudf::logic_error If `search_key` type does not match the element type in `lists`
  * @throw cudf::logic_error If `search_key` is of a nested type, or `lists` contains nested
  * elements (LIST, STRUCT)
+ * @throw cudf::data_type_error If `search_keys` type does not match the element type in `lists`
```

Comment:
Please remove from line 129 to line 131.
Lines 130-131 are left from previous changes (now all overloads of `index_of` support nested types).

Reply:
Done